src/main/java/com/softserve/academy/spaced/repetition/service/MailService.java

```diff
@@ -63,4 +94,34 @@ private String getFreeMarkerTemplateContent(Map <String, Object> model) {
         }
         return "";
     }
+
+
+    private String getChangePasswordTemplateContent(Map <String, Object> model) {
+        StringBuilder content = new StringBuilder();
+        try {
+            content.append(FreeMarkerTemplateUtils.processTemplateIntoString(
+                    freemarkerConfiguration.getTemplate("changePasswordMailTemplate.txt"), model));
+            return content.toString();
+        } catch (IOException e) {
+            logger.log(e.getClass().getName());
```

Comment:
Please change LOGGER initialization to 

`private static final Logger LOGGER = LoggerFactory.getLogger(MailService.class);`

as in CardController. 

Please log error the right way https://www.slf4j.org/faq.html#exception_message, something like

`LOGGER.error("Could generate email content.", e);`